conjure-java-undertow-runtime/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This makes it so that Mockito can mock final classes (needed to mock `HttpServerExchange`).